packages/core/package.json

```
@@ -47,7 +47,7 @@
     "events": "3.3.0",
     "isomorphic-unfetch": "3.1.0",
     "lodash.isequal": "4.5.0",
-    "uint8arrays": "3.1.0"
+    "uint8arrays": "^5.1.0"
```

```suggestion
    "uint8arrays": "5.1.0"
```
All versions should be fixed

---

my bad, just fixed them now